### ISSUE TITLE
css-loader doesn't work anymore with v0.15.1

### ISSUE THREAD

**Issue by gitawego** [webpack/css-loader#84]
after updated to v0.15.1 from v0.14.x, css loader doesn't work any more.
![error1](https://cloud.githubusercontent.com/assets/438120/8249446/5a391216-166a-11e5-8915-b423abbd1413.jpg)

here is the config:

```js
const sassLoaders = [
  "css",
  "sass?" + ["outputStyle=expanded",
    "includePaths[]=" + nodeModulesPath,
    "includePaths[]=" + bowerComponentsPath].join("&")
].join("!");
```
later on
```js
{
        test: /\.scss$/,
        loader: ExtractTextPlugin.extract("style-loader", sassLoaders, {
          publicPath: './'
        })

      },
```

**Comment by gitawego** [comment]
I've noticed in file /node_modules/css-loader/lib/processCss.js, wether I use sourcemap or not, the variable ``inputMap`` is always undefined.. and it crashes postcss. 
```js
var result = pipeline.process(inputSource, {
		from: options.from,
		to: options.to,
		map: {
			prev: inputMap,
			sourcesContent: true,
			inline: false,
			annotation: false
		}
	});
```
but if I define inputMap as '' when it's undefined, postcss will pass.
```js
var result = pipeline.process(inputSource, {
		from: options.from,
		to: options.to,
		map: {
			prev: inputMap || '',
			sourcesContent: true,
			inline: false,
			annotation: false
		}
	});
```

**Comment by lucianlature** [comment]
Confirmed.

**Comment by stevewillard** [comment]
Same issue here.

**Comment by chriserik** [comment]
The solution above works for me as well.

**Comment by bsr203** [comment]
same issue. Here is my stack (slightly different)

```
ERROR in /usr/local/lib/~/css-loader!/usr/local/lib/~/sass-loader?outputStyle=expanded!./src/client/style/index.scss
Module build failed: TypeError: Cannot read property 'sections' of null
    at new SourceMapConsumer (/usr/local/lib/node_modules/css-loader/node_modules/postcss/node_modules/source-map/lib/source-map/source-map-consumer.js:23:21)
    at PreviousMap.consumer (/usr/local/lib/node_modules/css-loader/node_modules/postcss/lib/previous-map.js:37:34)
    at new Input (/usr/local/lib/node_modules/css-loader/node_modules/postcss/lib/input.js:42:28)
    at parse (/usr/local/lib/node_modules/css-loader/node_modules/postcss/lib/parse.js:17:17)
    at new LazyResult (/usr/local/lib/node_modules/css-loader/node_modules/postcss/lib/lazy-result.js:54:47)
    at Processor.process (/usr/local/lib/node_modules/css-loader/node_modules/postcss/lib/processor.js:30:16)
    at processCss (/usr/local/lib/node_modules/css-loader/lib/processCss.js:168:24)
    at Object.module.exports (/usr/local/lib/node_modules/css-loader/lib/loader.js:21:15)
 @ ./src/client/style/index.scss 4:14-225 13:2-17:4 14:20-231

```

**Comment by schepelin** [comment]
Thanks! Your fix works for me

**Comment by HarrisRobin** [comment]
Confirmed. I simply downgraded to 0.14x instead of modifying library code.

**Comment by phatshambler** [comment]
Same here, fix works.

**Comment by dreampulse** [comment]
I have the same problem..

**Comment by mmahalwy** [comment]
same here. I get it from `extract-text-plugin` which requires any latest version.

```
ERROR in ./~/extract-text-webpack-plugin/loader.js?{"omit":1,"extract":true,"remove":true}!./~/style-loader!./~/css-loader!./~/sass-loader?outputStyle=expanded!./~/bootstrap-sass-loader/bootstrap-sass-styles.loader.js!./bootstrap-sass.config.js
Module build failed: ModuleBuildError: Module build failed: TypeError: Cannot read property 'sections' of null
    at new SourceMapConsumer (/quran/node_modules/css-loader/node_modules/postcss/node_modules/source-map/lib/source-map/source-map-consumer.js:23:21)
    at PreviousMap.consumer (/quran/node_modules/css-loader/node_modules/postcss/lib/previous-map.js:37:34)
    at new Input (/quran/node_modules/css-loader/node_modules/postcss/lib/input.js:42:28)
    at Object.parse [as default] (/quran/node_modules/css-loader/node_modules/postcss/lib/parse.js:17:17)
    at new LazyResult (/quran/node_modules/css-loader/node_modules/postcss/lib/lazy-result.js:54:42)
```

**Comment by cameronjroe** [comment]
0.14x works for me. Been debugging this all day.. : - (

**Comment by pthorn** [comment]
Same here. Reverted to 0.14.5.

**Comment by qimingweng** [comment]
+1 0.15 breaks for me, downgraded to 0.14

**Comment by ruifortes** [comment]
also adding sourceMap parameter seems to work
see here http://stackoverflow.com/questions/30932592/sass-loader-not-working-in-webpack

**Comment by maslianok** [comment]
+1 same issue

**Comment by mhuggins** [comment]
+1, downgrading the dependency to "~1.4.0" fixed the issue for me.

**Comment by gitawego** [comment]
new version (v0.15.2) still has this problem.....

**Comment by rlucha** [comment]
+1, same here

**Comment by Wicus** [comment]
yes same issue

**Comment by vn38minhtran** [comment]
+1 same issue

**Comment by jblack10101** [comment]
+1 same here

**Comment by markdalgleish** [comment]
This issue should now be fixed in v0.15.3.
